### PR TITLE
KIALI-2840 Rename Info tab to Overview

### PR DESCRIPTION
**Pull request by aljesusg**
JIRA: [KIALi-2893](https://issues.jboss.org/browse/KIALI-2893)

Continue of https://github.com/kiali/kiali-ui/pull/1171
- Rename info tab to Overview like design https://github.com/kiali/kiali-design/issues/116
- Clean code, remove empty ```<div></div>```

cc @abonas @skondkar

### CODE REVIEW

File: src/pages/ServiceDetails/ServiceDetailsPage.tsx [hunk 1]
```diff
@@ -262,15 +262,9 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
...
-              </NavItem>
-              <NavItem eventKey="traffic">
-                <div>Traffic</div>
-              </NavItem>
-              <NavItem eventKey="metrics">
-                <div>Inbound Metrics</div>
-              </NavItem>
+              <NavItem eventKey="info">Overview</NavItem>
+              <NavItem eventKey="traffic">Traffic</NavItem>
+              <NavItem eventKey="metrics">Inbound Metrics</NavItem>
```
Comment (abonas) [inline src/pages/ServiceDetails/ServiceDetailsPage.tsx]: in another file in this PR Inbound metrics key is in_metrics and here it is just metrics. isn't it a bit confusing/not consistent?

File: src/pages/WorkloadDetails/WorkloadDetailsPage.tsx [hunk 2]
```diff
@@ -183,27 +183,17 @@ class WorkloadDetails extends React.Component<RouteComponentProps<WorkloadId>, W
...
-              </NavItem>
-              <NavItem eventKey="in_metrics">
-                <div>Inbound Metrics</div>
-              </NavItem>
-              <NavItem eventKey="out_metrics">
-                <div>Outbound Metrics</div>
-              </NavItem>
+              <NavItem eventKey="info">Overview</NavItem>
+              <NavItem eventKey="traffic">Traffic</NavItem>
+              <NavItem eventKey="logs">Logs</NavItem>
```
Comment (abonas) [inline src/pages/WorkloadDetails/WorkloadDetailsPage.tsx]: is the order here OK? in other places it's traffic, then metrics, here it is traffic, logs, metrics.
(I know this existed prior to this PR, and doesn't need to be fixed as part of this PR)

File: src/pages/ServiceDetails/ServiceDetailsPage.tsx (hunk 1 above)
Comment (aljesusg) [reply src/pages/ServiceDetails/ServiceDetailsPage.tsx]: Totally agree I am think about move all of this in a component to reuse it and avoid this, but we should do it in another PR

Comment (abonas) [reply src/pages/ServiceDetails/ServiceDetailsPage.tsx]: no objections from my side

**Pull request merged**